src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
+- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api).
+- Tags are case sensitive.
```

Comment:
It's ok here but I would consider skipping it in 3.5

Reply:
Just to make sure if I get it right - you mean skipping both points completely?
To confirm - tags will not be case sensitive in 3.5, yes?

And also...I cannot find this comment but you wrote that you would avoid pushing the user toward AA when they are browsing API docs - Can I ask why cause I was not aware of that 😅

Reply:
this comment was only about case sensitivity... and in 3.5 it's more complicated, there is another PR that describes changes there so maybe the description in 3.5 should be based on that

as for AA aversion... if a developer comes to create their own integration and reads API docs we should show him how to do things API-way, not asking him to use "some GUI app" to skip API requests

Reply:
Concerning case sensitivity:

1. Keep the "- Tags are case sensitive." sentence in v3.2-v3.4.
2. As for v3.5, my assumption is that the v3.5 Tag Thread method checks if a tag already exists before tagging a thread, so we should adjust the v3.5 description. Something like, "In this method, letter case needs to be respected for tag names." This would suggest that it's only for this method, and not all tag-related methods in v3.5. 
@gluszczyk, correct me if my story's made up :)

Reply:
I have pushed new changes with the v3.5 as I think it will be better to discuss when seeing it:)

Reply:
@ale-kacperczyk in v3.5, **Create Tag** is the first method that ignores letter case when checking if the tag already exists, and the sentence "Tags are case sensitive" becomes **mostly** true 😉 so it might be better to state case-sensitivity explicitly in tag methods

maybe the sentence can explicitly state which field needs to respect the letter case?
>In this method, `tag` is case sensitive.

Grammarly fixed case sensitivity into case-sensitivity 😉 so what's with this BrE vs AmE you talked about in the other PR? 🤔

Reply:
@gluszczyk It may be due to your Grammarly settings (maybe, idk). I had consulted "case sensitive vs case-sensitive" with our in-house editor and he recommended to stick to "case sensitive" without a hyphen.